pkg/kubelet/deviceplugin/device_plugin_stub.go

```diff
@@ -86,6 +87,30 @@ func (m *Stub) Stop() error {
 	return m.cleanup()
 }
 
+// Register registers the device plugin for the given resourceName with Kubelet.
+func (m *Stub) Register(kubeletEndpoint, resourceName string) error {
```

Comment:
Does this method need to be exported?

Reply:
Right now, this stub is not used outside but it could be if we want to test integration with device plugin, at which point we will want to have this API exported.